test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/EmailControllerSpec.scala

```diff
@@ -283,34 +283,6 @@ trait EmailControllerSpec[JourneyType <: EmailJourneyType] extends ControllerSpe
         )
         checkIsRedirect(result, checkYourInboxCall)
       }
-
-    "strip out spaces in emails" in {
-      val emailWithSpaces    = " a @ b  "
-      val emailWithoutSpaces = "a@b"
-      val emailToBeVerified  =
-        EmailToBeVerified(Email(emailWithoutSpaces), id, verified = false, hasResentVerificationEmail = true)
-
-      inSequence {
-        mockAuthWithNoRetrievals()
-        mockGetSession(
-          sessionDataWithValidJourneyStatus
-            .copy(emailToBeVerified = Some(emailToBeVerified))
-        )
-        mockEmailVerification(
-          Email(emailWithoutSpaces),
-          expectedName,
-          verifyEmailCall(id),
-          AcceptLanguage.EN
-        )(
-          Right(EmailVerificationRequested)
-        )
-      }
-
-      val result: Future[Result] = performAction(
-        Seq("email" -> emailWithSpaces, "resendVerificationEmail" -> "true")
-      )
-      checkIsRedirect(result, checkYourInboxCall)
-    }
   }
 
```

Comment:
Please add unit test for additional validation

Reply:
Added test case for new validation.